assets/js/modules/analytics/common/profile-select.test.js

```diff
@@ -38,28 +38,30 @@ apiFetchMock.mockImplementation( ( ...args ) => {
 
 const setupRegistry = ( { dispatch } ) => {
 	const { id, webPropertyId, accountId } = fixtures.propertiesProfiles.profiles[ 0 ];
+	const propertyID = webPropertyId;
 	dispatch( STORE_NAME ).setAccountID( accountId );
 	dispatch( STORE_NAME ).setPropertyID( webPropertyId );
 	dispatch( STORE_NAME ).setProfileID( id );
-	dispatch( STORE_NAME ).receiveProfiles( fixtures.propertiesProfiles.profiles );
+	dispatch( STORE_NAME ).receiveProfiles( fixtures.propertiesProfiles.profiles, { propertyID } );
 };
 
 const setupRegistryWithExistingTag = ( { dispatch } ) => {
 	const existingTag = {
 		accountID: fixtures.accountsPropertiesProfiles.profiles[ 0 ].accountId,
 		propertyID: fixtures.accountsPropertiesProfiles.profiles[ 0 ].webPropertyId,
 	};
+	const propertyID = existingTag.propertyID;
 	const { id } = fixtures.propertiesProfiles.profiles[ 0 ];
 	dispatch( STORE_NAME ).setAccountID( existingTag.accountID );
 	dispatch( STORE_NAME ).setPropertyID( existingTag.propertyID );
 	dispatch( STORE_NAME ).setProfileID( id );
-	dispatch( STORE_NAME ).receiveProfiles( fixtures.accountsPropertiesProfiles.profiles );
+	dispatch( STORE_NAME ).receiveProfiles( fixtures.accountsPropertiesProfiles.profiles, { propertyID } );
 	dispatch( STORE_NAME ).receiveExistingTag( existingTag.propertyID );
 };
 
 const setupEmptyRegistry = ( { dispatch } ) => {
 	dispatch( STORE_NAME ).setSettings( {} );
-	dispatch( STORE_NAME ).receiveProfiles( [] );
+	dispatch( STORE_NAME ).receiveProfiles( [], {} );
```

Comment:
This doesn't really work - let's pass any property ID here so that it is valid. Since we don't pass anything here, `propertyID` will be `undefined`, resulting in the empty array not being assigned correctly.

Reply:
Fixed